app/controllers/comments_controller.rb

```diff
@@ -1,20 +1,33 @@
 class CommentsController < ApplicationController
   before_action :authenticate_user!
```

Comment:
Thanks Ken! If I put it under the authenticate_user! it still doesn't work, but if I put it above it,  it solves the problem.  (I think I understand why that is, but maybe a little clarification from you wouldn't hurt?)